zstacklib/zstacklib/gpu/vendors/nvidia.py

```
@@ -351,6 +354,131 @@ def pre_detach_from_host(cls):
+        """Find /dev/nvidia{N} device path for a given PCI address.
+
+        Uses nvidia-smi to get the authoritative index-to-PCI mapping.
+
+        Args:
+            pci_address: Normalized PCI address (e.g., "0000:34:00.0").
+
+        Returns:
+            Device path like "/dev/nvidia0", or None if not found.
+        """
+        try:
+            r, o, e = bash_roe(
```

Ruff (0.15.9)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)